doc/reference/tooling/tcm/index.rst

```diff
@@ -0,0 +1,36 @@
+:noindex:
+:fullwidth:
+
+..  _tcm:
+
+Tarantool Cluster Manager
+=========================
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    |tcm_full_name| is a part of the `Enterprise Edition <https://www.tarantool.io/compare/>`_.
+
+|tcm_full_name| (|tcm|) is a web-based visual tool for configuring, managing, and
+monitoring Tarantool EE clusters. It provides a GUI for working with clusters
+and individual instances, from monitoring their state to executing commands interactively
+in an instance's console.
+
+|tcm| is a standalone application included in the Tarantool Enterprise Edition
+distribution package. It is shipped as an executable that is ready to run on Linux
+and macOS platforms.
+
+|tcm| works only with Tarantool EE clusters that are configured through `etcd <https://etcd.io/>`_.
+When you create or edit a cluster's configuration in |tcm|, it sends the saved
+configuration to etcd. This ensures consistent and reliable configuration storage.
+Learn more about :ref:`storing configuration in etcd <configuration_etcd>`.
+
+A single |tcm| installation can manage multiple Tarantool EE clusters.
+A convenient switch in the web UI lets you select the desired cluster in one click.
```

Comment:
This paragraph sounds a bit too marketingish to me :) At the beginning, we already said about `managing EE clusters` and this might be enough IMO.

Reply:
Thanks, rephrased. AFAIR our PM wanted to highlight that one TCM can manage multiple clusters, so I'm leaving it, although in a more technical wording.